install_ffmpeg.sh

```diff
@@ -20,15 +30,20 @@ if [ ! -e "$HOME/x264/x264" ]; then
   cd "$HOME/x264"
   # git master as of this writing
   git checkout 545de2ffec6ae9a80738de1b2c8cf820249a2530
-  ./configure --prefix="$HOME/compiled" --enable-pic --enable-static
+  ./configure --prefix="$HOME/compiled" --enable-pic --enable-static ${HOST_OS:-}
   make
   make install-lib-static
 fi
 
+EXTRA_FFMPEG_FLAGS=""
+if [ $(uname) != "Darwin" ]; then
+  EXTRA_FFMPEG_FLAGS="--enable-cuda --enable-cuvid --enable-nvenc"
+fi
+
 if [ ! -e "$HOME/ffmpeg/libavcodec/libavcodec.a" ]; then
-  git clone -b n4.1 https://git.ffmpeg.org/ffmpeg.git "$HOME/ffmpeg" || echo "FFmpeg dir already exists"
+  git clone --depth 1 -b n4.1 https://git.ffmpeg.org/ffmpeg.git "$HOME/ffmpeg" || echo "FFmpeg dir already exists"
   cd "$HOME/ffmpeg"
-  ./configure --disable-programs --disable-doc --disable-sdl2 --disable-iconv \
+  ./configure ${TARGET_OS:-} --disable-programs --disable-doc --disable-sdl2 --disable-iconv \
```

Comment:
Is TARGET_OS needed for Windows? (Are we cross-compiling?)

Reply:
Correct. We're not cross-compiling but the configure scripts for x264 and ffmpeg have trouble autodetecting that we're doing an mingw64 build. There might be some way for me to better specify this in the Docker environment, but this seems to work too.